### ISSUE TITLE
Sorry if not an issue

### ISSUE THREAD

**Issue by fraygos** [noobot/noobot#86]
But I am trying to customize Noobot by adding our own commands. The wiki talks about noobot.custom
https://github.com/noobot/noobot/wiki/Customising-Noobot
But I cannot seem to find it.
Sorry if dumb question, just not sure what I am missing here.

**Comment by Workshop2** [comment]
Hey @fraygos 
Take a look at https://github.com/noobot/Noobot.Examples  for a couple of examples.

You need to write new Middleware and register them in the configuration class, e.g. 
https://github.com/noobot/Noobot.Examples/blob/master/src/Noobot.Examples.ConsoleService/Configuration/ExampleConfiguration.cs

**Comment by sophiaaar** [comment]
Hello,

I'm also confused about this, and the examples repo isn't helping. Do I need to create a new folder called Noobot.Custom? Do I copy files over from Core?

**Comment by Workshop2** [comment]
No problem, I will write some documentation soon. Sorry for the delay

**Comment by sophiaaar** [comment]
I ended up kind of copying the FlickrMiddleware in the Noobot.Toolbox and adapting it for the API I wanted to use

**Comment by AshleyPoole** [comment]
I think the documentation really needs some work ASAP. The readme, examples and wiki don't tie up which is leading to confusion by multiple people trying to consume the Noobot framework.

**Comment by Workshop2** [comment]
@AshleyPoole I completely agree. I have quickly deleted the old bits of wiki and added links to point to the example Repo until I find some time to re-write the Wiki area.
